src/apps/themes/domain.py

```diff
@@ -60,3 +63,9 @@ class PublicTheme(_ThemeBase, PublicModel):
 
 class Theme(ThemeCreate):
     id: PositiveInt
+
+
+class ThemeQueryParams(BaseQueryParams):
+    public: bool | None = None
+    allow_rename: bool | None = None
+    creator: int | None = None
```

Comment:
I think that if you define the boolean fields you can define False as a default value instead of having multiple types for the field. What do you think?

Reply:
I can not define it as false, because if default value is false it will add filter to query `public is false`, to ignore fields I used `None`